doc/cli_vs_yum.rst

```diff
@@ -117,14 +117,15 @@ Also see related Fedora bugzillas `982947
 
 .. _skip_if_unavailable_default:
 
-============================================
- ``skip_if_unavailable`` enabled by default
-============================================
-
-The important system repos should never be down and we see the third-party repos
-down often enough to warrant this change. Note that without this setting and
-without an explicit ``skip_if_unavailable=True`` in the relevant repo .ini file
-YUM immediately stops on a repo error, confusing and bothering the user.
+====================================================
+ ``skip_if_unavailable`` could be enabled by default
+====================================================
+
+In some distribution DNF is shipped with ``skip_if_unavailable=True`` in
+:ref:`DNF configuration file <conf_ref-label>` file. The reason for the change are the third-party
+repository that down often. Note that without this setting and without an explicit
+``skip_if_unavailable=True`` in the relevant repo .ini file YUM immediately stops on a repo error,
+confusing and bothering the user.
```

Comment:
Please use this sentence:
> ... The reason for the change is that third-party repositories may often be unavailable. ...

And then for:
> ... Note that without this setting and without an explicit ``skip_if_unavailable=True`` in the relevant repo .ini ...

What is _"this setting"_? To me it seems _"this setting"_ == `skip_if_unavailable=True`, so it seems to me this sentence is unnecessarily complex and confusing?